store/postgres/src/models.rs

```diff
@@ -17,3 +23,61 @@ pub struct EntityTable {
     #[sql_type = "VarChar"]
     pub event_source: String,
 }
+
+#[derive(Clone, Debug, PartialEq, AsExpression)]
+pub struct SqlValue(Value);
+
+impl SqlValue {
+    pub fn new_array(values: Vec<Value>) -> Vec<Self> {
+        values.into_iter().map(|value| SqlValue(value)).collect()
+    }
+}
+
+impl ToSql<Bool, Pg> for SqlValue {
+    fn to_sql<W: Write>(&self, out: &mut Output<W, Pg>) -> serialize::Result {
+        match self.0 {
+            Value::Bool(ref b) => <bool as ToSql<Bool, Pg>>::to_sql(&b, out),
+            _ => unimplemented!(),
```

Comment:
`unimplemented` isn't the right description here, a custom panic message would be better.